src/greentea/gtea/report_api.py

```diff
@@ -160,11 +174,14 @@ def exporter_testcase_text(test_result_ext):
 
 
 def exporter_testcase_junit(test_result_ext, test_suite_properties=None):
-    """! Export test results in JUnit XML compliant format
-    @param test_result_ext Extended report from Greentea
-    @param test_spec Dictionary of test build names to test suite properties
-    @details This function will import junit_xml library to perform report conversion
-    @return String containing Junit XML formatted test result output
+    """Export test results in JUnit XML compliant format.
+
+    Args:
+        test_result_ext: Extended report from Greentea.
+        test_spec: Dictionary of test build names to test suite properties.
+
+    Returns:
+        String containing Junit XML formatted test result output.
     """
     from junit_xml import TestSuite, TestCase
```

Comment:
I think we should create a task to remove the function scoped imports: it's not good form to import a bunch of stuff in a public api function, especially if it's going to be called more than once (we'll run the import code again on each call). It's a small cost to import these things at file scope, and a cost we only need to pay once at "import time". The function scoped imports are misguided attempts at micro-optimisation imo.